cfe_internal/enterprise/CFE_hub_specific.cf

```diff
@@ -48,6 +48,20 @@ bundle common cfe_internal_hub_vars
       "SSLCertificateKeySize" string => "4096";
       "SSLCertificateDaysValid" string => "3650";
 
+      # Determine the version of PHP that is used
+
+      "php_version"
+        string => ifelse( fileexists( "$(sys.workdir)/httpd/modules/libphp7.so" ), "7",
+                          fileexists( "$(sys.workdir)/httpd/modules/libphp5.so" ), "5"
```

Comment:
need a comma here I think, after "5" and before "UNKNOWN"

Reply:
Yes indeed
good catch.